client/post-editor/editor-drawer/featured-image.jsx

```diff
@@ -49,7 +49,7 @@ class EditorDrawerFeaturedImage extends Component {
 		const { translate, site, post, isDrawerHidden } = this.props;
 
 		return (
-			<Accordion title={ translate( 'Featured Image' ) }>
+			<Accordion title={ translate( 'Featured Image' ) } temporaryExpand={ isDrawerHidden }>
```

Comment:
Reiterating my previous comment on the naming. Yes, the temporal dimension makes for a very weird UI description here. Indeed, what is time when you contemplate a pure `return <Component temporaryFoo={}>`? 🤔